src/data/__tests__/deleteAll.test.ts

```diff
@@ -3,13 +3,13 @@ import { setupDeleteSuccess } from './deleteOne.test';
 
 describe('deleteAll', () => {
   test('calls the openapi delete endpoint, passing deleteAll with target namespace', async () => {
-    const { VoaProvider, VOA } = setupDeleteSuccess(undefined);
+    const { DataProvider, DPA } = setupDeleteSuccess(undefined);
```

Comment:
Renamed these across tests to keep things consistent with the new `DataPlaneApi` class.